package.json

```diff
@@ -0,0 +1,76 @@
+{
+  "name": "nhsuk-tools-chart-components",
+  "version": "1.0.1",
```

Comment:
should be version 1.0.0

Reply:
in fact, I'd put this at 0.0.1 until we're fully done with implementation and proved it out then we move it to 1.0.0